### DISCUSSION TITLE
V2 model biding, unique rule and ignore id

### DISCUSSION THREAD

**Discussion by jcergolj** [livewire/livewire#1590] [4. General]
I have an issue with the new model binding and how rules can be applied. Looks like ignore method cannot be used.
```
class EditUserComponent extends Component
{
    /** @var \App\Models\User */
    public User $user;

    /**
     * Validation rules.
     *
     * @return array
     */
    protected function rules()
    {
        return [
            'user.email' => [
                Rule::unique('users')->ignore($this->user->id)
            ]
        ];
    }
```
and the error: _$user must not be accessed before initialization_ 

Any ideas? Thanks.

**Comment by zepfietje** [comment]
@jcergolj, I ran into something similar recently. A PR to fix this issue has just been submitted: https://github.com/livewire/livewire/pull/1652.
